linera-storage/src/lib.rs

```diff
@@ -173,6 +177,54 @@ pub trait Storage: Sized {
         information: &NetworkDescription,
     ) -> Result<(), ViewError>;
 
+    /// Returns a map of the committees for the given epochs.
+    async fn committees_for(
+        &self,
+        mut epochs: BTreeSet<Epoch>,
+    ) -> Result<BTreeMap<Epoch, Committee>, ViewError> {
+        let read_committee = async |committee_hash| -> Result<Committee, ViewError> {
+            let blob_id = BlobId::new(committee_hash, BlobType::Committee);
+            let committee_blob = self
+                .read_blob(blob_id)
+                .await?
+                .ok_or_else(|| ViewError::NotFound(format!("Blob not found: {}", blob_id)))?;
```

Comment:
`NotFound` already prints `entry does not exist in memory`, so this could just say:

```suggestion
                .ok_or_else(|| ViewError::NotFound(format!("blob {}", blob_id)))?;
```

Actually, why "in memory"? Should probably be "in storage"?